### PR TITLE
GUVNOR-3236: Rearrange menus structure

### PR DESCRIPTION
**Pull request by paulovmr**

### CODE REVIEW

**Comment by paulovmr** [comment]
Part of an ensemble:
* https://github.com/AppFormer/uberfire/pull/798
* https://github.com/kiegroup/kie-wb-common/pull/1053
* https://github.com/kiegroup/drools-wb/pull/572
* https://github.com/kiegroup/jbpm-form-modeler/pull/133
* https://github.com/kiegroup/jbpm-wb/pull/846
* https://github.com/kiegroup/optaplanner-wb/pull/202
* https://github.com/kiegroup/kie-wb-distributions/pull/578

File: jbpm-wb-showcase/src/main/java/org/jbpm/workbench/client/ShowcaseEntryPoint.java [hunk 1]
```diff
@@ -77,7 +76,7 @@ public ShowcaseEntryPoint(final Caller<AppConfigService> appConfigService,
     @Override
     protected void setupMenu() {
 
-        final AbstractWorkbenchPerspectiveActivity defaultPerspective = menusHelper.getDefaultPerspectiveActivity();
+        final PerspectiveActivity defaultPerspective = menusHelper.getDefaultPerspectiveActivity();
```
Comment (ederign) [inline jbpm-wb-showcase/src/main/java/org/jbpm/workbench/client/ShowcaseEntryPoint.java]: permission

Comment (ederign) [inline jbpm-wb-showcase/src/main/java/org/jbpm/workbench/client/ShowcaseEntryPoint.java]: optional, just make sure that is checked on distributions.

Comment (paulovmr) [inline jbpm-wb-showcase/src/main/java/org/jbpm/workbench/client/ShowcaseEntryPoint.java]: All permission check for home is done on Uberfire now.

**Comment by paulovmr** [comment]
jenkins retest this

**Comment by paulovmr** [comment]
jenkins retest this

**Comment by paulovmr** [comment]
jenkins retest this

**Pull request merged**